src/coreclr/interpreter/compiler.cpp

```diff
@@ -31,9 +34,55 @@ static const InterpType g_interpTypeFromStackType[] =
     InterpTypeI,        // F
 };
 
+// Used by assertAbort
+thread_local ICorJitInfo* gInterpJitInfoTls = nullptr;
```

Comment:
A nit - the convention in the coreclr is to use `t_` prefix for thread local variables.